server/BUILD

```diff
@@ -72,7 +72,6 @@ native_java_libraries(
 
         # External dependencies from Maven
         "@maven//:com_google_code_findbugs_jsr305",
-        "@maven//:com_google_guava_guava",
```

Comment:
Actually without guava, project fails to build.

Reply:
What is the error?

Generally speaking Guava methods can be trivially implemented using our own code.

Reply:
@alexjpwalker It fails to build the GRPC generated code, there's nothing we can do to implement.